tests/modules/entrezdirect/xtract/test.yml

```diff
@@ -0,0 +1,21 @@
+- name: entrezdirect xtract test_entrezdirect_xtract_assembly
+  command: nextflow run ./tests/modules/entrezdirect/xtract -entry test_entrezdirect_xtract_assembly -c ./tests/config/nextflow.config  -c ./tests/modules/entrezdirect/xtract/nextflow.config
+  tags:
+    - entrezdirect/xtract
+    - entrezdirect
+  files:
+    - path: output/entrezdirect/test_assembly.txt
+      md5sum: 17073db7c14890aef30f5e3f2d003d32
+    - path: output/entrezdirect/test_assembly.xml
+      md5sum: ddbf36702303ef805a19f81754eb9c92
```

Comment:
Is this produced by xtract? If not I would remove it for both tests.

Reply:
No, it is only the input it takes, so it could be removed.